Example/Swift/ViewController.swift

```diff
@@ -140,6 +140,7 @@ class ViewController: UIViewController {
 //        bulletinManager.hidesHomeIndicator = true
 //        bulletinManager.backgroundColor = .blue
 
+        reloadManager()
```

Comment:
This should be placed above the block of comments, since `reloadManager` resets the properties someone might have uncommented in the block.